README-trust.md

```diff
@@ -0,0 +1,118 @@
+Trust module
+============
+
+Description
+-----------
+
+The trust module allows to ensure presence and absence of a domain trust.
+
+Features
+--------
+
+* Trust management
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipagroup module.
+
+Requirements
+------------
+
+**Controller**
+
+* Ansible version: 2.8+
+
+**Node**
+
+* Supported FreeIPA version (see above)
+* samba-4
+* server-trust-ad
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
+
+Example playbook to create a one-way trust:
+
+```yaml
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: create a one-way trust to the windows.local domain
+    ipatrust:
+      realm: windows.local
+      admin: Administrator
+      password: secret_password
+      state: present
+```
+
+Example playbook to create a two-way trust using a shared-secret:
+
+```yaml
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: create a two-way trust to the windows.local domain
```

Comment:
States are ensured by Ansible. Use "ensure trust is present".

Reply:
done